daemons/based/based_notify.c

```diff
@@ -237,26 +250,43 @@ cib_replace_notify(const char *origin, xmlNode *update, int result,
     }
 
     if (add_updates != del_updates) {
-        crm_info("Replaced: %d.%d.%d -> %d.%d.%d from %s",
-                 del_admin_epoch, del_epoch, del_updates,
-                 add_admin_epoch, add_epoch, add_updates,
-                 pcmk__s(origin, "unspecified peer"));
-    } else if (diff != NULL) {
-        crm_info("Local-only Replace: %d.%d.%d from %s",
-                 add_admin_epoch, add_epoch, add_updates,
-                 pcmk__s(origin, "unspecified peer"));
+        do_crm_log(log_level,
+                   "Replaced (client: %s%s%s): %d.%d.%d -> %d.%d.%d from %s "
+                   "(%s)",
+                   client_name,
+                   ((call_id != NULL)? ", call: " : ""), pcmk__s(call_id, ""),
```

Comment:
Looking at this convention, I think it would be easier to read as simply `" call "`

BTW there seems to be some redundancy with the "Replaced" message in `cib_process_replace()`

Reply:
I'm thinking like `Client CLIENTNAME call N replaced: ...`

Reply:
Or maybe `Replaced CIB generation ... with ... from client C call N (ORIGIN)`

Reply:
ack, btw my example didn't format correctly at first because it got parsed as html. Your statements still apply.

Reply:
I think I'll leave the redundancy for now and create a task for it. In `cib_process_replace()`, we return without logging if `cib_xpath` is set. However, we don't enter `cib_replace_notify()` unless particular sections are changed.